Cargo.toml

```diff
@@ -11,6 +11,8 @@ repository = "https://github.com/lurk-lab/lurk-rs"
 
 [dependencies]
 anyhow = "1.0.69"
+base64 = "0.13.1"
```

Comment:
I suspect it might suffice to put some of these in dev-dependencies.

Reply:
`base64` was transfered over from `fcomm` when I moved over the parameter caching part of the library, so I don't think it'd work as a dev dependency.

Reply:
My point is you don't need things in both places: dev and non-dev